### DISCUSSION TITLE
Add support for Threads profiles

### DISCUSSION THREAD

**Discussion by AmyJeanes** [community/community#60271] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Similar to how Mastodon and other general social media profiles were added as part of [this discussion](https://github.com/orgs/community/discussions/5720) I would like to see proper support for Threads.

[Threads](https://en.wikipedia.org/wiki/Threads_(app)) is Instagram's new Twitter-like social media service and has gained 30 million users in the first 24 hours already.

What I would like to see is the Threads icon in the profile UI along with the shortened username instead of the full URL.

Current behaviour:
![image](https://github.com/community/community/assets/2363642/395c5f0b-f005-46c7-87ea-6d73f42e2a50)

Terrible mock-up:
![image](https://github.com/community/community/assets/2363642/575da728-b1f2-4d7b-b79d-49b5eded91ae)

Another thing worth noting is that Threads is intending to have Fediverse support in the coming months, so not sure how that might affect the integration given that GitHub already supports fedi profiles, maybe it will all 'just work' when they release it? It may be worth showing the full username e.g. `@mattjeanes@threads.net` due to this as well.

**Comment by adityaruplaha** [comment]
This would be extremely useful, as Threads is emerging as a popular alternative to Twitter. An organisation I'm a part of is interested in making the switch right now.

**Comment by ryebytescode** [comment]
Yeah I need this!

**Comment by pjpscriv** [comment]
@MattJeanes I've created a Chrome extension to do this while we wait for GitHub to get to it https://chromewebstore.google.com/detail/github-social-icons/ledidlccnpfooopagmkgihgibfbnbbnp 😊

![image](https://github.com/user-attachments/assets/943a4921-825e-44fb-bdf7-addcaa26fbdb)
